{{cookiecutter.project_slug}}/docs/contributing.md

```diff
@@ -1,5 +1,17 @@
 # Contributing
 
+In order to be able to contribute, it is important that you understand
+the project layout.
+{% if cookiecutter.project_layout == "src" -%}
```

Comment:
Instead of 

```
This project uses the *src layout*, which means that the code you want
to import is placed in a subdirectory, usually called src.
```

I would say:

```
{% if cookiecutter.project_layout == "src" -%}
This project uses the *src layout*, which means that the package code is located
at `./src/{{ cookiecutter.package_slug }}`.
{% else -%}
This project uses the *flat layout*, which means that the package code is located
at `./{{ cookiecutter.package_slug }}`.
{% endif %}
For my information, check the official documentation:
https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/
```